### PR TITLE
Tic Tac Toe Complete

### PR DESCRIPTION
**Pull request by xyngabo**

### CODE REVIEW

File: ticTacToe.py [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-    #                                                                       #
-    # Hint: you can follow the same process that was done in the textbook.  #
-    #########################################################################
-
+    print(board['top-L'] + '|' + board['top-M'] + '|' + board['top-R'])
+    print('-+-+-')
+    print(board['mid-L'] + '|' + board['mid-M'] + '|' + board['mid-R'])
+    print('-+-+-')
+    print(board['low-L'] + '|' + board['low-M'] + '|' + board['low-R'])
+printBoard(theBoard)
```
Comment (halfaro1) [inline ticTacToe.py]: @xyngabo good job, but you don't need this line, there's already code in line 33 that will take care of this

File: ticTacToe.py [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
-    # TO DO #################################################################
-    # Write code in this function that checks the tic-tac-toe board          #
-    # to determine if the player stored in variable 'player' currently      #
-    # has a winning position on the board.                                  #
-    # This function should return True if the player specified in           #
-    # variable 'player' has won. The function should return False           #
-    # if the player in the variable 'player' has not won.                   #
-    #########################################################################
-    
+    return ((board['top-L'] == player and board['top-M'] == player and board['top-R'] == player) or
```
Comment (halfaro1) [inline ticTacToe.py]: @xyngabo are you copying and pasting this from somewhere else? while collaboration is encouraged, please make sure you understand your own work fully. Let me know if you need help with it.

Comment (xyngabo) [inline ticTacToe.py]: No, but I was lost initially and found another website that explained it and used their method.  I was very sick that week but I am not falling behind or anything.

**Comment by halfaro1** [comment]
@xyngabo alright just double checking. Good job.

**Pull request closed without merging**